src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}

```

Comment:
Follow the checklist, please 
<img width="1110" alt="image" src="https://user-images.githubusercontent.com/99403198/216305171-3831f987-91a7-4c1d-a2bf-956e0fe88e42.png">